util/ibex_config.py

```diff
@@ -214,6 +214,46 @@ def get_config_file_location():
     return os.environ.get('IBEX_CONFIG_FILE', _DEFAULT_CONFIG_FILE)
 
 
+def parse_config(config_name, config_filename=get_config_file_location()):
+    """Parses the selected config file and returns selected config information.
+
+    Arguments:
+
+        config_name: Name of the chosen Ibex core config
+
+        config_filename: Name of the configuration filename to be parsed
+
+    Returns: the chosen Ibex config as a YAML object.
+
+    Raises a ConfigException if there are any error while parsing the YAML.
+
+    Raises a FileNotFoundError if there are errors opening the chosen config file.
+    """
+    try:
+        config_file = open(config_filename)
+        config_dicts = get_config_dicts(config_file)
+
+        if config_name not in config_dicts:
+            print('ERROR: configuration',
+                  config_name,
+                  'not found in',
+                  config_filename,
+                  file=sys.stderr)
```

Comment:
I know this isn't your code but, if you're touching it anyway, could you turn this into a call to `format` to tidy things up? Something like `print('ERROR configuration {!r} not found in {!r}.'.format(config_name, config_filename), file=sys.stderr)`

Reply:
done